docs/operating-eck/eck-permissions.asciidoc

```diff
@@ -71,6 +71,7 @@ These permissions are needed by the Service Account that ECK operator runs as.
 |PodDisruptionBudget|policy|no|Ensuring update safety for Elasticsearch. Check link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-pod-disruption-budget.html[docs] to learn more.
 |StorageClass|storage.k8s.io|yes|Validating storage expansion support. Check link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-volume-claim-templates.html#k8s_updating_the_volume_claim_settings[docs] to learn more.
 |coreauthorization.k8s.io|SubjectAccessReview|yes|Controlling access between referenced resources. Check link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-restrict-cross-namespace-associations.html[docs] to learn more.
+|apiextensions.k8s.io|CustomResourceDefinition|yes|Preventing accidental CRD Deletion. Check <<{p}-enabling-crd-prevention-webhook,docs>> to learn more.
```

Comment:
This style of doc link is different than the previous, as I couldn't get a link such as `link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-webhook.html#k8s_enabling_the_crd_prevention_webhook[docs]` to work, as the docs build would say "no link found". Any pointers, or is it a non-issue?

Reply:
I might be wrong but I think the way you did it with the template is more correct. I think we want to avoid hard-coding absolute URLs in our docs where possible.